e2e/e2e_test.go

```diff
@@ -205,3 +204,49 @@ func TestDeleteJobComplete(t *testing.T) {
 	assert.Nil(err)
 	assert.True(configMapDeleted)
 }
+
+func TestCreateStateWithWorkspace(t *testing.T) {
+
+	assert := assert.New(t)
+	err := e.createStateWithWorkspace()
+	if err != nil {
+		logrus.Fatalf("Creating terraform state failed: %s", err.Error())
```

Comment:
logrus.Fatalf exists test immediately with exit code 1. Please consider something like `assert.Failf(t,....)` and fix all occurences.

Reply:
This is fine and desired result. All environment is temporary and tests are run sequentially

Reply:
But if the test fails so hard, then following tests will not start, right?